multidim-interop/src/generator.ts

```diff
@@ -139,27 +134,35 @@ function buildSpec(containerImages: { [key: string]: string }, { name, dialerID,
                 environment: {
                     version: dialerID,
                     transport,
-                    muxer,
-                    security,
                     is_dialer: true,
                     ip: "0.0.0.0",
+                    ...(!!muxer && { muxer }),
+                    ...(!!security && { security }),
                     ...extraEnv,
                 }
             },
             listener: {
+                // Add init process to be PID 1 to proxy signals. Rust doesn't
+                // handle SIGINT without this
```

Comment:
We could do that. But as far as I know, running init as PID is considered a good practise so I feel a bit uneasy rust-libp2p being called out as the bad actor here :)

Reply:
lol, it is the only one that *needs* this :P